src/frontend/src/test-e2e/views.ts

```diff
@@ -183,37 +183,22 @@ export class RecoveryMethodSelectorView extends View {
   }
 
   async getSeedPhrase(): Promise<string> {
-    // This tries to read the recovery phrase by first copying it to the clipboard.
-
-    await this.copySeedPhrase();
-
-    // Our CSP policy prevents us from directly reading the clipboard.
-    // Instead, we mock user input to paste the clipboard content in textarea element and
-    // read the element's value.
-
-    // First, create a new textarea element where the phrase will be pasted
-    await this.browser.execute(() => {
-      const elem = document.createElement("textarea");
-      elem.setAttribute("id", "my-paste-area");
-      document.body.prepend(elem);
-    });
-
-    // Select the element and mock "Ctrl + V" for pasting the clipboard content into said element
-    await this.browser.$("#my-paste-area").click();
-    await this.browser.keys(["Control", "v"]);
-
-    // Read the element's value and clean up
-    const seedPhrase = await this.browser.execute(() => {
-      const elem = document.querySelector(
-        "#my-paste-area"
-      ) as HTMLTextAreaElement;
-      // NOTE: we could also query the value with wdio's $(..).getValue(), but since we have
-      // the element here might as well.
-      const seedPhrase = elem.value!;
-      elem.remove();
-      return seedPhrase;
-    });
-
+    // Ideally, we could press the copy button on the page and the read the
+    // clipboard. However, this is not possible due to the content security
+    // policy.
+    // The alternative solution of simulating key presses does not work either,
+    // since chromium does not allow to interact with the clipboard via keyboard
+    // shortcuts when run in headless mode (which is the only mode accepted by CI).
+    // For the lack of a better solution, we read the seed phrase from the DOM.
+
+    const seedPhrase = (await this.browser.execute(() =>
+      Array.from(document.querySelectorAll(".c-list--recovery-word"))
+        // @ts-ignore
```

Comment:
I think you can replace with `Array.from(document.querySelectorAll(...)).map`

Reply:
This is already done?

Reply:
I thought so to. Maybe the tab was out of sync...